MIP41/mip41.md

```diff
@@ -142,29 +142,54 @@ MIP41c5 subproposals must use the template located at [MIP41c5-Subproposal-Templ
+The Facilitators of other Core Units may elect an alternative Interim Facilitator within 72 hours of the initial Facilitator Offboarding using the Interim Facilitator Election Process. In the event an alternative Interim Facilitator is elected, they immediately replace the current Interim Facilitator, if one exists.
 
-The incentive to volunteer as an Interim Facilitator is the compensation tied to a Core Unit unwinding plan. In the event of a missing, failed succession, or headless core unit, the incentive to volunteer as an Interim Facilitator is a transfer of the former Facilitator's compensation for an interim timeline.
+The incentive to volunteer as an Interim Facilitator is the compensation tied to a Core Unit unwinding plan. In the event of a missing, failed succession, or headless core unit, the incentive to volunteer as an Interim Facilitator is a transfer of the former Facilitator's compensation during the interim period.
 
-If a Core Unit has no active Facilitators as a result of involuntary offboarding and has no active budget, no Interim Facilitator will be appointed unless there are winding down activities that require one.
+#### Interim Facilitator Responsibilities And Powers
+
+An Interim Facilitator is empowered to serve in this role for a maximum of two months. 
+
+In the event of a Core Unit offboarding, the Interim Facilitator is responsible for overseeing and executing the unwinding process. 
+
+In the event of a Facilitator offboarding, the Interim Facilitator is responsible for both:
+* Ensuring the functional continuity of the Core Unit during the interim period.
+* Making a good faith effort to deliver one or more viable Facilitator Onboarding subproposals for the Core Unit to Maker Governance.
+
+The Interim Facilitator is empowered to use the Facilitator Governance Powers defined in MIP41c2 to keep existing operational processes running smoothly during the interim period.
+
+#### Interim Facilitator Election Process
+
+The Interim Facilitator Election Process assumes the existence of one or more permanent Governance Facilitators. If this is not the case, MIP41c6 must be considered non-binding.
+
+An Interim Facilitator may be elected by a majority of Core Units voting in favour of a given individual. The individual must be willing to take the role to be eligible for election. Each Core Unit may cast a single vote for or against each candidate. In the event of a Core Unit containing multiple facilitators, they must come to consensus on their Core Unit's votes before communicating them publicly.
 
-#### The Fallback Decision Process
+In the absence of succesful Interim Facilitator selection or election within 72 hours, one or more permanent facilitator onboarding proposals may be made under the next weekly governance cycle. This overrides the normal rules relating to the feedback and frozen periods.
```

Comment:
"next weekly governance cycle" may be impractical if this all kicks off on a weekend for example, make it slightly more vague to allow a delay until the next cycle if necessary

```suggestion
In the absence of succesful Interim Facilitator selection or election within 72 hours, one or more permanent facilitator onboarding proposals may be expedited through the weekly governance cycle. This overrides the normal rules relating to the feedback and frozen periods.
```